### ISSUE TITLE
Component Accordion: aria-expanded attribute is not set correctly

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#1515]
Description  
---
1. On the trigger button of the Accordion header, an `aria-expanded` attribute ist set but its value is not. It must change the value between `true` and `false` regarding of its state.
2. Missing `href` attribute of `p-link-pure` on the `compact` example.

To Reproduce
---
1. Go to accordion component
2. Check with AXE DevTools
3. See error

Expected behavior
---
`aria-expanded` values are set correctly

Test Environment
---
Platform/Browser/Version: all
